tests/src/junit/java/net/neoforged/neoforge/unittest/transfer/TransactionTests.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.unittest.transfer;
+
+import net.neoforged.neoforge.transfer.transaction.Transaction;
+import net.neoforged.neoforge.transfer.transaction.snapshots.IndexedIntSnapshot;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
Why is the test order relevant here? It doesn't seem to be, so I'd remove the annotation.